packages/atclient/include/atclient/atnotification.h

```diff
@@ -4,7 +4,7 @@
 extern "C" {
 #endif
 
-#include "atclient/cjson.h"
+#include "atcommons/json.h"
```

Comment:
moved json header to only be in atcommons, atclient now depends on atcommons.

Also renamed from cjson to json to be generic to any json library.